isis/src/qisis/objs/AdvancedTrackTool/AdvancedTrackTool.cpp

```diff
@@ -569,7 +569,30 @@ namespace Isis {
         Cube *cCube = cvp->cube();
         int iTrackBand = -1;
 
-        if(cCube->hasTable(TABLE_MOSAIC_SRC)) {
+        if(cCube->hasGroup("Tracking")) {
+          PvlGroup trackingGroup = cCube->group("Tracking");
+          trackingGroup.findKeyword("FileName")[0];
+          FileName trackingCubeFileName(trackingGroup.findKeyword("FileName")[0]); 
+          Cube trackingCube(trackingCubeFileName);
```

Comment:
I'm worried about opening and closing the tracking cube everytime this method gets called, which is everytime the mouse moves.